File: TM 15 C#-Basics/C#-Basics/Control Statements/program1.cs
// Tek Raj Joshi
// Superset ID: 1368453

using System;
class Program1
{
public static getFibonacci()  
{  
    int a = 0, b = 1, c = 0;  
    Console.Write("{0} {1}", a,b);  
    for (int i = 2; i <= 20; i++)  
    {  
        c= a + b;  
        Console.Write(" {0}", c);  
        a= b;  
        b= c;  
    }  
}
}

File: TM 2 Data Structure/Data Structure/String/program1.py
# Tek Raj Joshi
# Superset ID: 1368453

input_string = input("Enter a string: ")
upper_count, lower_count = 0, 0
for i in input_string:
    if i.islower():
        lower_count += 1
    else:
        upper_count += 1

print(upper_count, lower_count)